newspack-sacha/inc/child-color-patterns.php

```diff
@@ -38,10 +38,12 @@ function newspack_sacha_custom_colors_css() {
 		.entry-meta .byline a:visited,
 		.entry .entry-meta a:hover,
 		.accent-header,
+		.widgettitle,
```

Comment:
This should be targeted to the sidebar (as well as the footer below) so it's not picked up by any other widget spaces:
```suggestion
		#secondary .widgettitle,
```